src/amo/components/App/index.js

```diff
@@ -70,7 +75,31 @@ export function getErrorPage(status: number | null) {
 export class AppBase extends React.Component<Props> {
   scheduledLogout: TimeoutID;
 
-  static defaultProps = {
+  static defaultProps: {|
+    _addChangeListeners: (
+      callback: ({|
+        canUninstall: boolean,
+        guid: string,
+        needsRestart: boolean,
+        status: $Values<{|
+          onDisabled: string,
+          onDisabling: string,
+          onEnabled: string,
+          onEnabling: string,
+          onInstalled: string,
+          onInstalling: string,
+          onUninstalled: string,
+          onUninstalling: string,
+        |}>,
+      |}) => Promise<void>,
+      mozAddonManager: MozAddonManagerType,
+      _?: {| _log: any |},
+    ) => any,
+    _navigator: Navigator | null,
+    authTokenValidFor: any,
+    mozAddonManager: any | { ... } | void,
```

Comment:
This was added by the codemod and somehow I missed it. It cannot just be `MozAddonManagerType` because it can also contain `{}`, so I changed it to `MozAddonManagerType | Object`.